internal/pkg/instrumentation/manager.go

```diff
@@ -207,7 +210,7 @@ func (m *Manager) cleanup(target *process.TargetDetails) error {
 func (m *Manager) Close() error {
 	m.done <- true
 	m.wg.Wait()
-	err := <-m.closingErrors
+	err := <-m.runningErrors
```

Comment:
nit: Do I see correctly that `Run` and `Close` are returning the same error? If so then I find it awkward. Maybe we should rename it to `Stop` and not return an error? Especially that it is not even handeled:

https://github.com/RonFed/opentelemetry-go-instrumentation_fork/blob/36a6ebe436445344912565ce87a1dcfba9b17eca/instrumentation.go#L141-L146

Reply:
Good point, I forgot to handle that error in `Instrumentation.Close`.
Currently, there are 2 types of errors in the Run function which we can return:
* "Loading" errors which can occur if we fail to load an eBPF program at the start of `Run`.
* Closing errors that can occur while cleaning up the probes.

What do you think about returning the first type of error from `Run` and the second from `Close`?

Reply:
It will be for sure better